src/test/functional/tests/addUser.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect, test } from '@playwright/test';
+
+import { adminUserTestData } from '../data';
+import { LoginPage } from '../pages/LoginPage';
+import { PimPage } from '../pages/PimPage';
+
+let loginPage: LoginPage, pimPage: PimPage;
+
+test.describe('Adding a new employee', () => {
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    pimPage = new PimPage(page);
+    await loginPage.initialize();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
```

Comment:
Would be better to move it globalConfig and reuse storageState (please see playwright doc), but in either case - passwords shouldn't be committed.